internal/scheduler/common.go

```diff
@@ -24,6 +25,9 @@ type SchedulerResult struct {
 	// For each preempted job, maps the job id to the id of the node on which the job was running.
 	// For each scheduled job, maps the job id to the id of the node on which the job should be scheduled.
 	NodeIdByJobId map[string]string
+	// The Scheduling Context. Being passed up for metrics decisions made in scheduler.go and scheduler_metrics.go.
+	// Passing a pointer as the structure is enormous
+	SchedulingContextList []*schedulercontext.SchedulingContext
```

Comment:
```suggestion
	SchedulingContexts []*schedulercontext.SchedulingContext
```

Reply:
`SchedulingContexts` is consistent with the rest of the code base. And avoid putting the variable type (sort of, go has slices, not lists) in the name.

Reply:
Same comment for everywhere else.